src/synology_dsm/api/download_station/__init__.py

```diff
@@ -32,17 +37,50 @@ def update(self):
                 self._tasks_by_id[task_data["id"]] = SynoDownloadTask(task_data)
 
     # Global
+    def update_info(self):
+        """Update info about the Download Station instance."""
+        self._info = self._dsm.get(self.INFO_API_KEY, "GetInfo")["data"]
+
     def get_info(self):
         """Return general informations about the Download Station instance."""
-        return self._dsm.get(self.INFO_API_KEY, "GetInfo")
+        return self._info
+
+    def update_config(self):
```

Comment:
what is the intention of this dedicated update method?
Those the user is forced to do an manually initial update before config data can be accessed